code/game/machinery/computer/cloning.dm

```diff
@@ -526,7 +526,11 @@
 			has_bank_account = I.registered_account
 	if(isbrain(mob_occupant))
 		dna = B.stored_dna
-
+	
+	if(HAS_TRAIT(mob_occupant, NO_DNA_COPY))
+		scantemp = "<font class='bad'>Unable to locate genetic data.</font>"

```

Comment:
I like how you just copied the message below instead of just writing what qwerty told you to write.

Reply:
read above reviews

Reply:
Like the part you ignored?

Reply:
like the part which i did, and then qwerty said its still to specific?

Reply:
No, the part where he literally told you what to write and then instead of writing that you just copied the message from the next check instead.